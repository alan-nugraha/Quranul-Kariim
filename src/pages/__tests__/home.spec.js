import React from 'react';
import {render} from '@testing-library/react-native';
import Home from '../home';

describe('Home', () => {
  beforeEach(() => {});

  afterEach(() => {
    jest.clearAllMocks();
  });

  afterAll(() => {
    jest.restoreAllMocks();
  });

  test('should render Home correctly', () => {
    const {toJSON} = render(<Home />);

    expect(toJSON()).toMatchSnapshot();
  });
});
