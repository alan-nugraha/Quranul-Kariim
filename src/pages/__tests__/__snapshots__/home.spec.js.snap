// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Home correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F6F6F6",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#02C882",
        "borderBottomLeftRadius": 10,
        "borderBottomRightRadius": 10,
        "height": "12%",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Raleway-Bold",
          "fontSize": 20,
          "textAlign": "center",
        }
      }
    >
      Al - Quran
    </Text>
  </View>
  <RCTScrollView
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={0.5}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    showsVerticalScrollIndicator={false}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(255,255,255,0.5)",
        "bottom": 0,
        "flex": 1,
        "height": "100%",
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "width": "100%",
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#07A9F0"
      hidesWhenStopped={true}
      size={30}
    />
  </View>
</View>
`;
